src/utils/urlPaths.ts

```diff
@@ -48,6 +48,8 @@ export enum UrlPathTemplate {
   MyPageUserRoleAndHelp = '/my-page/profile/user-role-and-help',
   PrivacyPolicy = '/privacy-policy',
   Projects = '/projects',
+  Project = '/project',
+  ProjectWizard = '/project/:identifier/edit',
```

Comment:
Naturlig at disse er på samme base route, tenker jeg. 
feks noe sånt?
```suggestion
  ProjectsNew = '/projects/new',
  ProjectsEdit = '/projects/:identifier/edit',
```

Tenker det er greit å fjerne "Wizard" også, siden wizarden skal brukes for både opprett og endre (ev. bruke "Wizard" på begge i stedet)